devito/types/equation.py

```diff
@@ -157,6 +157,211 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Le(sympy.Le):
```

Comment:
we may wanna split that into `equation.py` and `relational.py`

Reply:
yes, this is a very good point. Let's do it